dependency/vault_pki_test.go

```diff
@@ -25,13 +25,18 @@ func init() {
 }
 
 func Test_VaultPKI_uniqueID(t *testing.T) {
-	d1, _ := NewVaultPKIQuery("pki/issue/example-dot-com", "/unique_1", nil)
+	d1, _ := NewVaultPKIQuery("pki/issue/example-dot-com", "/unique_1", nil, nil)
 	id1 := d1.String()
-	d2, _ := NewVaultPKIQuery("pki/issue/example-dot-com", "/unique_2", nil)
+	d2, _ := NewVaultPKIQuery("pki/issue/example-dot-com", "/unique_2", nil, nil)
 	id2 := d2.String()
 	if id1 == id2 {
 		t.Errorf("IDs should be unique.\n%s\n%s", id1, id2)
 	}
+	d3, _ := NewVaultPKIQuery("pki/sign/example-dot-com", "/unique_1", nil, nil)
+	id3 := d3.String()
+	if id1 == id3 {
+		t.Errorf("IDs should be unique.\n%s\n%s", id1, id3)
```

Comment:
```suggestion
		t.Errorf("IDs should be unique.\n%s\n%s\n%s", id1, id3, id2)
```
I think we need to compare all the 3?